test_team_tech_baddies.py

```diff
@@ -64,7 +64,27 @@ def test_in_vehicle_registration(self):
 
     def test_in_voter(self):
         """Test IN_VOTER functionality"""
+        begin = ['AAA']
+        ending = ['0000000']
+
+        # Positive Test Case
+        for b in begin:
+            for e in ending:
+                voter = ''.join([b,e])
+                # Check context score should be 0.75
+                result = analyze_text('My Voter ID is ' + voter, ['IN_VOTER'])
+                print(result)
+                self.assertEqual('IN_VOTER', result[0].entity_type)
+                self.assertEqual(0.75, result[0].score)
+
+                # Check no context
```

Comment:
voter is a context word for IN_VOTER according to the [opensource](https://github.com/microsoft/presidio/blob/cc31bb61982311d7e2ad7e3b5334da797cdb6f98/presidio-analyzer/presidio_analyzer/predefined_recognizers/in_voter_recognizer.py#L37) - so I don't see how this is a check for no context as stated in the comment.